tox.ini

```diff
@@ -102,6 +102,7 @@ commands =
     template: python3 bin/make_template {posargs}
     bddtests: sh bin/create-db lms_bddtests
     bddtests: behave {posargs:tests/bdd/}
+    typecheck: mypy lms
```

Comment:
We could eventually change from mypy to another checker, or run more than one...

Reply:
You'll have to add this to the cookiecutter or to `.cookiecutter/includes/tox/commands`